lib/widgets/total_amount.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import '../l10n/tr.dart';
+
+class TotalAmount extends StatelessWidget {
```

Comment:
Rename to `TotalAmountCard`, adding suffixes as `Card`, `Field`, `Button` makes the widget name more descriptive.